tools/lkl/lib/fs.c

```diff
@@ -102,7 +110,6 @@ int lkl_get_virtio_blkdev(int disk_id, uint32_t *pdevid)
 
 	ret = snprintf(sysfs_path + sysfs_path_len, sizeof(sysfs_path) - sysfs_path_len, "/%s/block",
 		       virtio_name);
-	free(virtio_name);
```

Comment:
Is it safe to do so? seems like d_name is pointing a memory that will be freed in lkl_closedir